conf/deployment/aws/ipi_3az_rhcos_3m_6w_arbiter.yaml

```diff
@@ -0,0 +1,25 @@
+---
+#  This configuration file is used for Metro DR Arbiter cluster deployment
+#  TODO: Add polarion ID
+DEPLOYMENT:
+  arbiter_deployment: true
+  allow_lower_instance_requirements: false
+  type: 'EBS'
+  local_storage: true
+  ocs_operator_nodes_to_label: 6
+  arbiter_zone: 'us-east-2c'
+  arbiter_autodetect: false
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 6
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
```

Comment:
I guess that we don't need `extra_disks: 2` here.

Reply:
But actually I will use this value to add more than only one EBS volume in AWS node for this. So in next commit I will use extra_disks ;)